Add new --ora-stuck-timeout parameter

pesse opened the pull request:
Reson for this that I have the assumption that catch is too greedy, leading to more abort- and retries than necessary.

pesse committed:
Add new --catch-ora-stuck parameter

Reson for this that I have the assumption that catch is too greedy, leading to more abort- and retries than necessary.

wiblash commented:
Is this already added to the latest release (3.1.8), since the mentioned parameter is not listed nor excepted as one.

pesse committed:
Merge branch 'develop' into bugfix/oracle_stuck_too_greedy

pesse committed:
New parameter --ora-stuck-timeout

Sets a timeout around Reporter creation and retries when not ready after a while. 0 = no timeout.

Fixes #199
Fixes #197
Fixes #196
Fixes #193
Fixes #191

pesse committed:
Fix failing test

pesse committed:
Fix another test

Yes I need my local setup again
